NuKeeper.Tests/Engine/Packages/PackageUpdateSortTests.cs

```diff
@@ -210,7 +212,10 @@ private static PackageUpdateSet UpdateSetFor(PackageIdentity package, DateTimeOf
 
         private static PackageInProject MakePackageInProjectFor(PackageIdentity package)
         {
-            var path = new PackagePath("c:\\temp", "folder\\src\\project1\\packages.config",
+            var isWindows = RuntimeInformation.IsOSPlatform(OSPlatform.Windows);
+            var baseDirectory = isWindows ? "c:\\temp" : "/temp";
```

Comment:
I think it's worth pulling these 2 lines out into a new static class & method, as they occur several times below, and they control how you interact with the OS.

Reply:
Hm, `c:\temp` will Work On My Machine and that's about it. Should be `Path.GetTempPath` `https://msdn.microsoft.com/en-us/library/system.io.path.gettemppath(v=vs.110).aspx`

Reply:
Ha, those are unit tests. This doesn't touch the actual file system, I have not `c:\temp` and the tests still work - they're building paths in memory. So, that's a starting point for parsing, not for actual file system access. Perhaps this could be more explicit by using `x:\imaginary-folder`?

Reply:
Ah, fine then.